site/docs/webrtc/voice-iw.md

```diff
@@ -0,0 +1,190 @@
+
+And by combining our WebRTC and programmable voice APIs, you can create composite call flows that meet your needs.
+
+The interaction between PV and WebRTC is initiated under application control from the PV side of the platform.
+
+![Summary of interconnection](../../static/img/webrtc-pv-summary.png "summary")
+
+To interconnect a WebRTC participant to the PV network capabilities, the customer application creates a WebRTC session and a participant for the session. This information is sent by your application to the PV platform which has been authorized to send a SIP invite to a WebRTC SIP URI. This URI contains a UUI header that contains a participant token which enables the system to securely identify and interconnect the voice capabilities and the WebRTC participants.
+
+Once an answer is received on this transaction, an interconnection is established between WebRTC and Voice.
+
+You are able to decide how the flow of the call will proceed based on Bxml verbs that are chosen on setting up the interconnection. Three call control verbs that can be used are: Transfer, Bridge, and Conference, each of which creates a different application experience
+
+| PV Verb used | Solution Characteristics summary |
+| Transfer | The simplest application behavior, with limited opportunity to invoke services on the PV call leg. |
+| Bridge | Suitable for single-leg interconnection where features might be required for the PV call leg, with slightly higher complexity required in the application |
+| Conference | Enables multiple participants on the PV side of the interconnection, and a full range of PV features and capabilities to be applied on the OV call leg. Additional costs are incurred for the use of the voice conference bridge. |
+
+These three interaction models are described in greater detail below.
+
+## TRANSFER
+
+The transfer verb creates a single PV call with two endpoints. You can set one end to WebRTC, and the other for a phone number
+
+![Transfer Verb Model](../../static/img/webrtc-pv-transfer.png "Transfer")
+
+An everyday use case for this method is when people use Bandwidth phone numbers as a proxy. If you run a business and wish to protect your personal number, then transfer would be a good option.
+
+Use of the Transfer Bxml Verb creates a single call that will typically be taken down from end to end when it is complete (or times out waiting for an answer). The call ends if the person you are transferring to does not respond. Also, any modifications to the call will take down the connection between WebRTC and the Voice Network.
+
+In general, the Transfer verb is used in inbound or outbound WebRTC/Voice calls to create the call from scratch when the call happens. Here’s a [general example](https://github.com/Bandwidth-Samples/webrtc-hello-world-ts) of how you can use the Transfer verb:
+
+Bandwidth's Voice API will hit a provisioned callback endpoint when you receive an incoming call
+
+```
```

Comment:
```suggestion
```js
```

Reply:
Add js to the code snippet opening on all samples to highlight syntax in the samples

Reply:
thanks for the cool trick - I've altered all instances.